postings/views.py

```diff
@@ -110,3 +111,52 @@ def delete(self, request, posting_id):
             posting_id = posting_id
         )
         return JsonResponse({'message': 'DELETE_LIKE'}, status=204)
+
+class WritingView(View):
+    
+    @authorize_user
+    def post(self, request):
+        try:
+            s3_client = boto3.client(
+                's3',
+                aws_access_key_id=AWS_ACCESS_KEY_ID,
+                aws_secret_access_key=AWS_SECRET_ACCESS_KEY
+            )
+            image = request.FILES['image']
+            my_uuid = str(uuid.uuid4())
+            s3_client.upload_fileobj(
+                image, 
+                AWS_STORAGE_BUCKET_NAME,
+                my_uuid,
+                ExtraArgs={
+                    "ContentType": image.content_type
+                })
+            image_url    = AWS_S3_CUSTOM_DOMAIN + my_uuid
+            
+            data         = json.loads(request.POST['info'])
+            housing_type = HousingType.objects.get(type=data['housing_type'])
+            size         = Size.objects.get(type=data['size'])
+            style        = Style.objects.get(type=data['style'])
+            back_color   = Color.objects.get(type=data['back_color'])
+
+            print('=======')
+            item_color   = Color.objects.get(type=data['item_color'])
+            text         = data['text']
+            
+            print('====123123123===')
+            Posting.objects.create(
+                user         = request.user,
+                housing_type = housing_type,
+                size         = size,
+                style        = style,
+                back_color   = back_color,
+                item_color   = item_color,
+                image        = image_url,
+                text         = text,
+                update_at    = datetime.datetime.now()
+                )
```

Comment:
```suggestion
                update_at    = datetime.datetime.now()
            )
```
update_at을 수동으로 넣은 이유를 코멘트로 남겨 주세요.